<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addCustomer</class>
 <widget class="QDialog" name="addCustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="acceptCancelButtonBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>261</width>
     <height>143</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="infoLayout">
      <item>
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="phoneNumberLabel">
        <property name="text">
         <string>Phone Number:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="emailLabel">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="businessLabel">
        <property name="text">
         <string>Busness:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="keyCustomerLabel">
        <property name="text">
         <string>Key:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="inputLayout">
      <item>
       <widget class="QLineEdit" name="inputName"/>
      </item>
      <item>
       <widget class="QLineEdit" name="inputPhoneNumber"/>
      </item>
      <item>
       <widget class="QLineEdit" name="inputEmail"/>
      </item>
      <item>
       <widget class="QLineEdit" name="inputBusiness"/>
      </item>
      <item>
       <widget class="QRadioButton" name="inputKeyCustomer">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>699</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="interestLevelLayout">
    <item>
     <widget class="QLabel" name="interestInterestLevel">
      <property name="text">
       <string>Interest Level:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="selectInterestLayout">
      <item>
       <widget class="QRadioButton" name="noInterestButton">
        <property name="text">
         <string>No Interest</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="lowInterestButton">
        <property name="text">
         <string>Low Interest</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="moderateInterestButton">
        <property name="text">
         <string>Moderate Interest</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="highInterestButton">
        <property name="text">
         <string>High Insterest</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="extremelyHighInterestButton">
        <property name="text">
         <string>Extremely High Interest</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>acceptCancelButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>addCustomer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptCancelButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>addCustomer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
